### PR TITLE
Update us_localbtv.m3u

### PR DESCRIPTION
**Pull request by Central302**
If You Using tvtv_us.xml , This Patch Will Add EPG Support (Sorry, Selected Stations Only)

### CODE REVIEW

**Review by dct-infra — DISMISSED** [review]

**Review by what-a — COMMENTED** [review]
Doesn't the tvg-id need the ".us" suffix?

**Review by BellezaEmporium — CHANGES_REQUESTED** [review]
You need the .us suffix on each tvg-id.

**Comment by Central302** [comment]
@LaneSh4d0w Thanks! Fixed

**Review by what-a — DISMISSED** [review]

File: streams/us_localbtv.m3u [hunk 1]
```diff
@@ -191,16 +191,16 @@ https://v-ny.theus6tv.tk/hls/41.2/playlist.m3u8
...
 #EXTVLCOPT:http-user-agent=Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/92.0.4515.131 Safari/537.36
 https://v-ny.theus6tv.tk/hls/7.2/playlist.m3u8
-#EXTINF:-1 tvg-id="" status="online" user-agent="Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/92.0.4515.131 Safari/537.36",WPXN-DT1 (Ion East)
+#EXTINF:-1 tvg-id="WPXNDT1.us" status="online" user-agent="Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/92.0.4515.131 Safari/537.36",WPXN-DT1 (Ion East)
 #EXTVLCOPT:http-user-agent=Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/92.0.4515.131 Safari/537.36
 https://v-ny.theus6tv.tk/hls/31.1/playlist.m3u8
-#EXTINF:-1 tvg-id="" status="online" user-agent="Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/92.0.4515.131 Safari/537.36",WPXN-DT2 (Bounce HD)
+#EXTINF:-1 tvg-id="WPXNDT2.ut" status="online" user-agent="Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/92.0.4515.131 Safari/537.36",WPXN-DT2 (Bounce HD)
```
Comment (LaneSh4d0w) [inline streams/us_localbtv.m3u:200]:
```suggestion
#EXTINF:-1 tvg-id="WPXNDT2.us" status="online" user-agent="Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/92.0.4515.131 Safari/537.36",WPXN-DT2 (Bounce HD)
```

**Review by what-a — APPROVED** [review]

**Review by BellezaEmporium — APPROVED** [review]

**Pull request merged**